pgpq.go

```diff
@@ -136,7 +137,19 @@ func (td *TaskDetails) scan(rows interface{ Scan(...interface{}) error }) error
 		&td.Namespace,
 		&td.Priority,
 		&td.Payload,
+		&td.NotBefore,
 		&td.CreatedAt,
 		&td.UpdatedAt,
 	)
 }
+
+// ----------------------------------------------------------------------------
+
+var unixZero = time.Unix(0, 0).UTC()
```

Comment:
You can just use time.Time{}

Reply:
What? You said to default to unix(0), now it's time.Time{}

If you're OK with this PR - just merge.

Reply:
And tag, just reminding

Reply:
Actually, looks like I have write access to this repo - let me merge && tag it.